Content.Shared/Climbing/Systems/ClimbSystem.cs

```diff
@@ -89,8 +89,10 @@ public override void UpdateBeforeSolve(bool prediction, float frameTime)
                 continue;
             }
 
-            var xform = _xformQuery.GetComponent(uid);
-            _xformSystem.SetLocalPosition(uid, xform.LocalPosition + comp.Direction * frameTime, xform);
+            if (!_containers.IsEntityInContainer(uid)) {
+                var xform = _xformQuery.GetComponent(uid);
+                _xformSystem.SetLocalPosition(uid, xform.LocalPosition + comp.Direction * frameTime, xform);
+            }
```

Comment:
Climbing should be stopped up the callstack and not added to a very expensive update function.

Reply:
moved cancel from the lerp to the init (UpdateBeforeSolve -> Climb)